src/views/virtualmachines/list/VirtualMachinesList.tsx

```diff
@@ -83,12 +84,22 @@ const VirtualMachinesList: React.FC<VirtualMachinesListProps> = ({ kind, namespa
           loaded={loaded}
           rowFilters={filters}
           onFilterChange={onFilterChange}
+          columnLayout={{
+            columns: columns?.map(({ id, title, additional }) => ({
+              id,
+              title,
+              additional,
+            })),
+            id: VirtualMachineModelRef,
+            selectedColumns: new Set(activeColumns?.map((col) => col?.id)),
```

Comment:
Do we need set here? It is possible to have same ids ?

Reply:
we need this because the selectedColumns is of type Set<string>, this avoids TS type error